### DISCUSSION TITLE
Bun don't have full support for Windows yet

### DISCUSSION THREAD

**Discussion by theobiabo** [oven-sh/bun#310] [Q&A] [answered]
After checking the bun official site I found out that it only supports sub window ( WSL )

**Comment by Bellisario** [comment]
No, currently the only way you can use on Windows it's to have WSL installed and run the linux version (it's about comparable to a virtual machine).
My problem, for example, is that I can't use WSL because I'm on Windows 7...

I just asked if and when will be available a Windows release here: #361

**Comment by chenrui333** [comment, marked as answer]
Windows support is on [Bun's roadmap][1] 

> ### Windows support
> - The HTTP client needs a Windows implementation for the syscalls
> - Bun needs test coverage for Windows filepath handling
> - All of bun's dependencies need to compile on Windows
> - Building JavaScriptCore needs to work on Windows and the JIT tiers need to work. I don't know what the current status of this is. WebKit's [bug tracker][2] suggests it may not have JITs enabled which will likely need some patches to fix it.


  [1]: https://github.com/Jarred-Sumner/bun/issues/159
  [2]: https://bugs.webkit.org/show_bug.cgi?id=145366
